MVP.md

```diff
@@ -11,3 +12,27 @@ In all of our investigations to-date, we have discovered one protocol which has
 
 The best solution we have found is a provider known as [Unlock Protocol](https://unlock-protocol.com/) which offers ERC-721 tokens that are fully configurable and can be implemented on a variety of common L1 and L2 blockchains.  From a pure functional perspective
 
+</details>
+
+# Problem = Signal 2 Noise Ratio
+
+<details><summary>The tokenomics of LexDAO are simple but the interactions are subtle:</summary>
+1. mmmber - annual fixed subscriptionin fiat but payable in stablecoins;
+   - working for membership - a loosely pegged conversion from base membership to LXCHAT, a timebank
+   - this timebank represents ~15 hrs of sweat as in-kind contribution in lieu of the annual membership subscription
+   - it is arguable whether these l'externs should have access to all guild channels given the semi-confidential nature of some projects but conceptually they have all access to the same platforms members have (discord, hackMD, github, etc)
+2. guest pass (discord only) - as a privilege members are permitted to issue guest passes for public to participate in special events within discord such as hackathons or study groups
+   - however most guest passes are automatically granted 4 wks via #introduction or by following the decision tree triage in #find-a-lawyer
+   - these guest passes are expoential backdown which means the 2nd renewal is at half the time
+   - they are automatically renewed if they submit a common which is "liked" by a LEETHodler or member performing the druid role (guest concierage)
+   - however, the same druids have power to revoke such guest privileges upon spam, behaviour contrary to community norms or upon complaint by regular member but which is appealable at @Ops total discretion
+3. LEETHodlers are a special case as there were certain historical discussions which grandfathered in certain privileges and retained as result of legacy (beware of cryptowizards for theur anger is slow but revenge is subtle.
```

Comment:
We need to determine the right way to grandfather status from this circle.  But I want LEETH to be claimable.  Both for new holders who earn the right to claim and old holders who have the right to claim.  Claim is the key mechanism here that needs to be structured.